thirtyinch/src/main/java/net/grandcentrix/thirtyinch/TiFragment.java

```diff
@@ -73,6 +75,11 @@ public P getPresenter() {
         return mDelegate.getPresenter();
     }
 
+    @Override
+    public Executor getUiThreadExecutor() {
+        return new UiThreadExecutor();
```

Comment:
It's harder to leak an Activity when the `Executor` isn't `static` or reused. Also it's a very small class, shouldn't cause problems